Automattic/jetpack
# Fix style inconsistency in ActionButton when Gutenberg plugin is active

#### Changes proposed in this Pull Request:

* This PR introduces CSS changes that fix some style inconsistencies in the `<ActionButton />` component that present when the Gutenberg plugin is active.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:






* The best place to see the original issue is on the Jetpack connection screen. However, it only seems to show following site disconnection.
* On your test site, first test the Disconnect Flow from the master branch.
* From the Jetpack dashboard, click on "Manage site connection" to open the disconnection flow.
![Screen Shot 2021-12-06 at 3 29 53 PM](https://user-images.githubusercontent.com/18016357/144920555-80065da6-ed42-4bc9-bb77-7a9e8bc35d12.png)

* Click on "Disconnect" from the first screen.
![Screen Shot 2021-12-06 at 3 30 04 PM](https://user-images.githubusercontent.com/18016357/144920597-b3a2e10f-85c8-41b7-8190-73a4795ea8a2.png)

* On the disconnect confirmation screen in the flow, click on "No thank you" to exit the flow and return to the connection screen. 
![Screen Shot 2021-12-06 at 3 30 14 PM](https://user-images.githubusercontent.com/18016357/144920625-c71959c4-4038-4dfe-9692-24f725a168c7.png)

* Notice here that the "Set up Jetpack" button is sized incorrectly, has incorrect border radius, text is misaligned, etc.
![Screen Shot 2021-12-06 at 3 38 54 PM](https://user-images.githubusercontent.com/18016357/144920639-4fc36be8-488d-4b82-9da5-9bdd5be57306.png)

* Now switch to this branch on your test site. Re-connect Jetpack and follow the same steps from above to disconnect and be redirected to the connection screen.
* The "Set up Jetpack" button on the connection screen should now appear correctly styled, as seen in the screenshot below.
![Screen Shot 2021-12-06 at 3 40 10 PM](https://user-images.githubusercontent.com/18016357/144920699-11f77dd4-c953-47b3-b317-7c4574857414.png)
